src/seeders/roles.js

```diff
@@ -0,0 +1,36 @@
+module.exports = {
```

Comment:
Why did you delete the `create-users` seeders

Reply:
I only renamed it, when sequelize required order.

Reply:
ok